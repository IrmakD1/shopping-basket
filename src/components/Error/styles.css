.error-banner {
    background-color: red;
    color: white;
    display: flex;
    justify-content: center;
    align-items: center;
    height: 3rem;
    margin-top: 0;
}