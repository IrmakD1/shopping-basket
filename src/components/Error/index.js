import React from 'react'
import './styles.css'

const Error = ({text}) => {
    return (
        <div className='error-banner'>
            <h4>{text}</h4>
        </div>
    )
}

export default Error