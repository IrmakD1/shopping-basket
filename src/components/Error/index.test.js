import React from 'react';
import renderer from 'react-test-renderer';
import Error from '.';

describe('Error Component', () => {
    let component;
    let tree;
  
    beforeEach(() => {
      component = renderer.create(
        <div>
          <Error text='Whooooopppsss'/>
        </div>
      );
      tree = component.toJSON();
    });
  
    test('it should render', () => {
      expect(tree).toMatchSnapshot();
    });
  });
  