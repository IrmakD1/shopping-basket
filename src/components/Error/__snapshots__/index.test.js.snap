// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component it should render 1`] = `
<div>
  <div
    className="error-banner"
  >
    <h4>
      Whooooopppsss
    </h4>
  </div>
</div>
`;
